interface-definitions/include/haproxy/logging.xml.i

```diff
@@ -0,0 +1,146 @@
+<!-- include start from haproxy/logging.xml.i -->
+<node name="logging">
+  <properties>
+    <help>Logging parameters</help>
+  </properties>
+  <children>
+    <tagNode name="facility">
```

Comment:
This looks like an exact copy of https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/syslog-facility.xml.i

I think it's better to include that file rather than copy its contents, since syslog facilities and levels are standardized and won't need custom options here.

Reply:
Thanks, I wanted to see a maintainer's opinion on this. 

I chose this direction as it would 
isolate system logging configuration changes from haproxy. This also allows for expanding additional logging options should someone want to add them.

Additionally there at least one difference in the level configuration. Although I can account for it in the Jinja template easily enough.

Reply:
Updated as suggested.